_data/team.yml

```diff
@@ -216,3 +216,14 @@
     <p>Now it's time to bring software craftmanship to Swift mobile, desktop and server.</p>
   twitter: guillemf
   github: guillemf
+
+- id: cyplo
+  name: Cyryl Płotnicki
+  image: /assets/img/custom/team/cyryl_plotnicki.jpg
+  role: Software Craftsman
+  fullDescription:
+    <p> Cyryl spent most of his professional life building and maintaining larger backend systems, mostly in .Net and JVM spaces. Gradually he became interested in the people management side of the work as well. Via this interest he found the software crafstmanship community and never looked back. He is focused on extracting the needs of the clients and helping them get the value out of the work quickly. </p>
+    <p> Interested making the systems, whether physical or virtual, more secure. Open source advocate with strong data privacy focus. He tries to try at least one new programming language a year, focusing on Rust lately. </p>
+    <p> Outside of work Cyryl likes to relax by taking part in the local hackerspace community, making electronics and machine parts.</p>
```

Comment:
Outside of work, Cyryl likes to relax by taking part in the local hackerspace community, making electronics and machine parts.